tests/components/input_datetime/test_init.py

```diff
@@ -619,7 +624,7 @@ async def test_ws_create(hass, hass_ws_client, storage_setup):
     assert resp["success"]
 
     state = hass.states.get(input_entity_id)
-    assert state.state == "1991-01-02 01:02:03"
+    assert state.state == "1991-01-02 1:02:03"
```

Comment:
Why do we want to change the hour format?

Reply:
We don't. Reverted.